apps/DemoApp/DemoAppUITests/DemoAppUITests.m

```diff
@@ -0,0 +1,41 @@
+//
+//  DemoAppUITests.m
+//  DemoAppUITests
+//
+//  Created by Nidhi Kulkarni on 10/27/20.
+//  Copyright © 2020 Mux, Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+@interface DemoAppUITests : XCTestCase
+
+@end
+
+@implementation DemoAppUITests
+
+- (void)setUp {
+    // Put setup code here. This method is called before the invocation of each test method in the class.
+
+    // In UI tests it is usually best to stop immediately when a failure occurs.
+    self.continueAfterFailure = NO;
+    
+    // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
+}
+
+- (void)tearDown {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+}
+
+- (void)testPlayVideo {
+    XCUIApplication *app = [[XCUIApplication alloc] init];
+    [app setLaunchEnvironment:@{@"ENV_KEY": @"is1368012c267a34rk0rhhr4u"}];
```

Comment:
Which org/env is this? I might suggest that we use a new environment in the same org that we use for [watch-this](https://github.com/muxinc/watch-this)

Instead of hard-coding, it might be nice to load this as an environment variable that gets configured into the build kite process.

Reply:
Ah good call. Yeah this is just my development org but I agree with changing it to the one here: https://github.com/muxinc/watch-this

> Instead of hard-coding, it might be nice to load this as an environment variable that gets configured into the build kite process.
This is a great idea, let me look into how to do this

Reply:
here is a view being sent to that environment: https://dashboard.mux.com/environments/ioi7qg/views/Y2Kd25ztDBjlLZiWJkS8NYSrajiWwmyAzZo